NineChronicles.Standalone.Tests/NineChronicles.Standalone.Tests.csproj

```diff
@@ -7,6 +7,168 @@
     <IsTestProject>true</IsTestProject>
   </PropertyGroup>
 
+  <ItemGroup>
+    <None Remove="Data\TableCSV\Character\CharacterLevelSheet.csv" />
+    <None Remove="Data\TableCSV\Character\CharacterSheet.csv" />
+    <None Remove="Data\TableCSV\Cost\EnhancementCostSheet.csv" />
+    <None Remove="Data\TableCSV\GameConfigSheet.csv" />
+    <None Remove="Data\TableCSV\Item\ConsumableItemRecipeSheet.csv" />
+    <None Remove="Data\TableCSV\Item\ConsumableItemSheet.csv" />
+    <None Remove="Data\TableCSV\Item\CostumeItemSheet.csv" />
+    <None Remove="Data\TableCSV\Item\EquipmentItemOptionSheet.csv" />
+    <None Remove="Data\TableCSV\Item\EquipmentItemRecipeSheet.csv" />
+    <None Remove="Data\TableCSV\Item\EquipmentItemSetEffectSheet.csv" />
+    <None Remove="Data\TableCSV\Item\EquipmentItemSheet.csv" />
+    <None Remove="Data\TableCSV\Item\EquipmentItemSubRecipeSheet.csv" />
+    <None Remove="Data\TableCSV\Item\ItemConfigForGradeSheet.csv" />
+    <None Remove="Data\TableCSV\Item\MaterialItemSheet.csv" />
+    <None Remove="Data\TableCSV\Quest\CollectQuestSheet.csv" />
+    <None Remove="Data\TableCSV\Quest\CombinationEquipmentQuestSheet.csv" />
+    <None Remove="Data\TableCSV\Quest\CombinationQuestSheet.csv" />
+    <None Remove="Data\TableCSV\Quest\GeneralQuestSheet.csv" />
+    <None Remove="Data\TableCSV\Quest\GoldQuestSheet.csv" />
+    <None Remove="Data\TableCSV\Quest\ItemEnhancementQuestSheet.csv" />
+    <None Remove="Data\TableCSV\Quest\ItemGradeQuestSheet.csv" />
+    <None Remove="Data\TableCSV\Quest\ItemTypeCollectQuestSheet.csv" />
+    <None Remove="Data\TableCSV\Quest\MonsterQuestSheet.csv" />
+    <None Remove="Data\TableCSV\Quest\QuestItemRewardSheet.csv" />
+    <None Remove="Data\TableCSV\Quest\QuestRewardSheet.csv" />
+    <None Remove="Data\TableCSV\Quest\TradeQuestSheet.csv" />
+    <None Remove="Data\TableCSV\Quest\WorldQuestSheet.csv" />
+    <None Remove="Data\TableCSV\RedeemCodeListSheet.csv" />
+    <None Remove="Data\TableCSV\RedeemRewardSheet.csv" />
+    <None Remove="Data\TableCSV\Skill\BuffSheet.csv" />
+    <None Remove="Data\TableCSV\Skill\EnemySkillSheet.csv" />
+    <None Remove="Data\TableCSV\Skill\SkillBuffSheet.csv" />
+    <None Remove="Data\TableCSV\Skill\SkillSheet.csv" />
+    <None Remove="Data\TableCSV\WeeklyArenaRewardSheet.csv" />
+    <None Remove="Data\TableCSV\WorldAndStage\StageDialogSheet.csv" />
+    <None Remove="Data\TableCSV\WorldAndStage\StageSheet.csv" />
+    <None Remove="Data\TableCSV\WorldAndStage\StageWaveSheet.csv" />
+    <None Remove="Data\TableCSV\WorldAndStage\WorldSheet.csv" />
+    <None Remove="Data\TableCSV\WorldAndStage\WorldUnlockSheet.csv" />

```

Comment:
I wonder what these `None` elements having `Remove` property do though each path seems included as `Content` in the below lines.

Reply:
It is auto-created by IDE. 
> By default, the sdk has something like <None Include="**/*"> (simplified) that is added (included) before your project's contents. But you don't want your file to be in the "None" set, but in the "EmbeddedResource" set.

[Reference](https://stackoverflow.com/a/49785443)